packages/abi-gen-templates/partials/tx.handlebars

```diff
@@ -1,4 +1,10 @@
 public {{languageSpecificName}} = {
+    /**
+     * Sends the transaction
```

Comment:
`Sends an Ethereum transaction executing this method with the supplied parameters. This is a read/write Ethereum operation and will cost gas.`